pkgs/applications/editors/emacs-modes/emacspeak/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, rsync, gnused, emacs, tcl, espeak-classic }:
+
+stdenv.mkDerivation rec {
+  pname = "emacspeak";
+  version = "50.0";
+
+  src = fetchFromGitHub {
+    owner = "tvraman";
+    repo = "emacspeak";
+    rev = version;
+    sha256 = "043mn3rr48gx2mi4rnkcvp09fwhs88yyqcszavkzvvd57mh4vd0h";
+  };
+
+  nativeBuildInputs = [rsync gnused] ;
+  buildInputs = [ emacs tcl espeak-classic ];
+  postPatch = ''
+    substituteInPlace servers/native-espeak/Makefile --replace '/usr/include/tcl$(TCL_VERSION)' ${tcl}/include
+    substituteInPlace servers/native-espeak/Makefile --replace 'LIBPARENTDIR = ''${PREFIX}/'  'LIBPARENTDIR = /'
+'';
+
+  preConfigure = ''
+    make config
+  '';
+
+  postBuild = ''
+  make espeak
+'';
+
+postInstall = ''
+  pushd servers/native-espeak
+  make DESTDIR=${placeholder "out"} install
+  popd
+  cp emacspeak-pkg.el run* $out/share/emacs/site-lisp/emacspeak/
+  echo -e "obsolete" > ex
+  rsync -a --exclude-from=ex lisp $out/share/emacs/site-lisp/emacspeak/
+  rsync -a --exclude-from=ex scapes $out/share/emacs/site-lisp/emacspeak/
+  rsync -a --exclude-from=ex utils $out/share/emacs/site-lisp/emacspeak/
+'';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/tvraman/emacspeak/;
+    description = "Emacs extension that provides spoken output";
+    license = licenses.gpl2;
+    maintainers = [];
```

Comment:
do you mind adding yourself as a maintainer? mostly just so you get notified if this gets bumped. Then you can verify that the software still works.